ONSdigital/rasrm-acceptance-tests
# Standalone account unlocking features

# Motivation and Context
Converts the respondent account locking and unlocking related scenarios to standalone so they can be run in parallel. Also deletes 3 scenarios from the 'respondent_unlocks_account.feature' because they are duplicates from another feature file.

# What has changed
- Convert account unlocking internal feature to standalone
- Convert forgotten password respondent feature to standalone
- Convert respondent unlocking account feature to standalone
- Convert respondent verification feature to standalone
- Delete duplicate tests in respondent unlocking account
- Add function to generate a year old timestamped email reset JWT

# How to test?
Run `make acceptance_tests`
All tests should pass and the listed feature should now not run in the sequential batch and run in the parallel batch.

# Links
https://trello.com/c/LraNWdPy/396-t396-convert-account-unlocking-features-to-standalone
